### ISSUE TITLE
Orchard.exe DataServiceContext TypeLoadException

### ISSUE THREAD

**Issue by kodeo** [OrchardCMS/Orchard#5152]
Fresh install of Orchard 1.8.1 via Web Platform Installer, on Windows 8.1 64. Everything appears fine creating and running a site via WebMatrix but on running bin\orchard.exe almost every command including "help commands" results in:

Could not load type 'System.Data.Services.Client.DataServiceContext' from assembly 'System.Data.Services.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.

Exception Details: System.TypeLoadException: Could not load type 'System.Data.Services.Client.DataServiceContext' from assembly 'System.Data.Services.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.

**Comment by Xeevis** [comment]
Works for me on 8.1 x64. `System.Data.Services.Client.dll` is GAC registered assembly so I'd say problem is coming from that angle.

First thing I'd do would be to check if assembly is properly registered. If you have **Visual Studio 2013** installed you can do that through **Developer Command Prompt**
Open run dialog with `Win + R` and paste in the following command

    %comspec% /k "C:\Program Files (x86)\Microsoft Visual Studio 12.0\Common7\Tools\VsDevCmd.bat"

In the resulting cmd write

    gacutil /l System.Data.Services.Client

It should display list like this

    The Global Assembly Cache contains the following assemblies:
      System.Data.Services.Client, Version=3.5.0.0, Culture=neutral, PublicKeyToken=
    b77a5c561934e089, processorArchitecture=MSIL
      System.Data.Services.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=
    b77a5c561934e089, processorArchitecture=MSIL

    Number of items = 2

**Comment by kodeo** [comment]
Thanks for the pointers Xeevis. 

The gacutil did produce exactly the above response, however I then found that the System.Data.Services.Client.dll under C:\Windows\Microsoft.NET\assembly\GAC_MSIL although exactly the same file size as the one under C:\Windows\Microsoft.NET\Framework64\v4.0.30319, was not byte for byte the same. I replaced the former with the latter and it's working now. 

Thanks again.
